src/icons/logo-icon-transparent.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from "react";
+
+interface LogoIconProps {
+  className?: string;
+}
+
+const LogoIconTransparent: FC<LogoIconProps> = ({ className = "" }) => {
```

Comment:
Instead of doing this you can declare it this way `const LogoIconTransparent = ({ className = '' }: LogoIconProps) => {`